CourseManager/Views/SampleListView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  SampleListView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-01.
+//
+
+import SwiftUI
+
+struct SampleListView: View {
+    var sampleCode = SampleWork(name: "profile", description: "profile view")
+    var cornerRadius = 20.0
+    var body: some View {
+        ZStack {
+            LinearGradient(gradient: Gradient(colors: [MainColors.accent, .blue, .purple]), startPoint: .top, endPoint: .bottomLeading).ignoresSafeArea()
+            ScrollView {
+                VStack {
+                    overlayContent
+                }
+            }
+        }
+    }
+
+    var overlayContent: some View {
+        ForEach(sampleWork, id: \.id) { sample in
+            VStack(alignment: .leading, spacing: 12) {
+                Text(sample.name)
+                    .font(.largeTitle.weight(.bold))
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                Text(sample.description)
+                    .font(.footnote)
+
+                Button {} label: {
+                    Text("View Example")
+                        .frame(maxWidth: .infinity)
+                }
+                .font(.headline)
+                .strokeStyle(cornerRadius: cornerRadius)
+                .buttonStyle(.angular)
+                .blendMode(.luminosity)
+                .tint(.accentColor)
+                .controlSize(.large)
+                .shadow(color: AppColors.shadow.opacity(0.2), radius: 5, x: 0, y: 1)
+            }
+            .padding(15)
+            .background(
+                Rectangle()
+                    .fill(.ultraThinMaterial)
+                    .mask(RoundedRectangle(cornerRadius: 30, style: .continuous))
+            )
+            .padding(.horizontal, 20)
+            .padding(.vertical, 8)
+        }
+    }
+}
+
+#warning("To be moved to separate file after Model content created. This is just a prototype.")
+
+struct SampleWork: Identifiable {
+    var id = UUID()
+    var name: String
+    var description: String
+}
+
+#warning("data below is just a filler, will be changing and making it more descriptive.")
+var sampleWork: [SampleWork] = [
```

Comment:
It is better to put such data in the following clause. Otherwise, your test data will end up in the production build

```
#if DEBUG

#endif
```